newIDE/app/src/AssetStore/AssetDetails.js

```diff
@@ -255,18 +250,18 @@ export const AssetDetails = ({
           </Column>
           <Column alignItems="center" justifyContent="center">
             <LeftLoader
-              isLoading={isBeingInstalled || (!asset && !error)}
+              isLoading={isBeingAddedToScene || (!asset && !error)}
               key="install"
             >
               <RaisedButton
-                primary
+                primary={!isAddedToScene}
                 label={
-                  isBeingInstalled ? (
-                    <Trans>Adding to my project...</Trans>
-                  ) : isAssetAdded ? (
-                    <Trans>Object added to my project</Trans>
+                  isBeingAddedToScene ? (
+                    <Trans>Adding to scene...</Trans>
+                  ) : isAddedToScene ? (
+                    <Trans>Add object again to my scene</Trans>
```

Comment:
"Add again"?
If it's not the 1st time, the user probably know what we're talking about.